conda_build/build.py

```diff
@@ -500,6 +501,8 @@ def test(m, verbose=True, channel_urls=(), override_channels=False):
 
     tmp_dir = join(config.croot, 'test-tmp_dir')
     rm_rf(tmp_dir)
+    time.sleep(2)  # give the OS some time to complete the rm_rf() command
```

Comment:
Is there an issue with more information on what the problem is? If anything I'd like to fix this in rm_rf (ideally by doing a check, rather than sleeping by an arbitrary amount of time).